freqtrade/exchange/exchange.py

```diff
@@ -1833,7 +1833,7 @@ async def _async_get_historic_ohlcv(self, pair: str, timeframe: str,
         )
         input_coroutines = [self._async_get_candle_history(
             pair, timeframe, candle_type, since) for since in
-            range(since_ms, until_ms or (arrow.utcnow().int_timestamp * 1000), one_call)]
+            range(since_ms if since_ms else 0, until_ms if until_ms else (arrow.utcnow().int_timestamp * 1000), one_call)]
```

Comment:
i'd rather not have that check included, as it's not really necessary.

you should be able to quite easily remove this by running `git rebase -i origin/develop` - and selecting "drop" on the middle commit.

Reply:
yeah no worries